src/main/java/sleepy/taskstorage/Storage.java

```diff
@@ -0,0 +1,65 @@
+package sleepy.taskstorage;
+
+import sleepy.tasks.Task;
+import sleepy.tools.Ui;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private File hardDiskFile;
+
+    /**
+     * Constructor for the Storage class.
+     *
+     * @param filePath filePath of the file used for storage.
+     */
+    public Storage(String filePath) {
+        try {;
+            hardDiskFile = new File(filePath);
+            hardDiskFile.createNewFile();
+        } catch (IOException i) {
+            Ui.printLine("Sleepy encountered a serious error! Please restart the application :(");
+        }
+    }
+
+    /**
+     * Returns the tasks read from the file as an ArrayList of strings.
+     *
+     * @return Tasks read from the file.
+     */
+    public ArrayList<String> readFile() {
+        ArrayList<String> readTasks = new ArrayList<>();
```

Comment:
Not sure if it was meant to be read(as in participle or present)Tasks. Personally, tasksRead might be clearer.

Reply:
I see! I would personally interpret `tasksRead` as a verb :sweat_smile: but I can totally see how it would mean the present participle, thanks for the input!